helper/validation/strings.go

```diff
@@ -0,0 +1,37 @@
+package validation
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strings"
+)
+
+// StringIsNotEmpty is a ValidateFunc that ensures a string is not empty or consisting entirely of whitespace characters
+func StringIsNotEmpty(i interface{}, k string) ([]string, []error) {
```

Comment:
This is already covered (sort of) by `NoZeroValues`, with the distinction that the type can be a string, or int, or float. I wouldn't be bothered if we accepted this too though as I would like to see the opposite added `StringIsEmpty`

Reply:
afaik `NoZeroValues` will not trigger on `property = "        "`

Reply:
You have the added Trim rule too which is good, we COULD probably add that to `NoZeroValues` :man-shrugging: . I think I like this method if we add the `StringIsEmpty` complement